src/analyzer.rs

```diff
@@ -182,6 +182,44 @@ impl ImportAssertions {
       _ => None,
     }
   }
+
+  pub fn is_known_none(&self, key: &str) -> bool {
```

Comment:
I'm confused by this method name because it does `!map.contains_key`. Maybe this should be `has(&self, key: &str)` then we can invert everything and do `!has(` when using it?

Reply:
Then `has()` would have to return true if the presence of associated value of that key is unknown. The method returns true if the given key is 100% known to be absent (that's where we treat it the assertion as `type: "js"`), and returns false if the set of keys is dynamic (can't check assertions) or the key is set to a value (we will use `get()` to check assertions).

Reply:
True. Maybe add a documentation comment on this?